dll/win32/shell32/CShellLink.cpp

```diff
@@ -271,6 +271,7 @@ CShellLink::CShellLink()
 
     m_sLinkPath = NULL;
     m_iIdOpen = -1;
+    m_bInitializing = FALSE;
```

Comment:
What's the purpose of this variable? (and the corresponding IsInit and SetInInit functions)
Because it doesn't seem to be used anywhere else in the code.

Reply:
In WM_INITDIALOG action, SetDlgItemText causes EN_CHANGE notification.
This notification can update the 'Changed' status of property sheet by WM_NOTIFY message (see the procedure).
We should prevent the status change.

Reply:
Made this variable a static variable in the procedure.

Reply:
That's not right. There's only one instance of a static variable so now multiple property dialogs can interfere with one another.